src/components/Headline.js

```diff
@@ -9,8 +9,9 @@ import { MarketStatus } from './MarketStatus';
 export const Headline = ({ stock }) => {
     const state = useSelector((state) => state)
     const dispatch = useDispatch()
-    const currency = stock=== null || undefined ? 'N/A' : stock.currency
-
+    const labels = Object.keys(stock).filter(key => key !== 'companyName' && key !== 'symbol').map(key => (
```

Comment:
What if `stock` is `null` or `undefined`? You're using `stock` in your conditional rendering below (line 23), so presumably it can be.

Reply:
Is ok to use 
`const labels = stock != undefined ? Object.keys(stock).filter(key => key !== 'companyName' && key !== 'symbol').map(key => (
        <li>{stock[key]}</li>
    )) : 'N/A'`

Reply:
Yep, but don't need the 'N/A'.
```
const labels = stock && Object.keys(stock)...
```